.github/workflows/release.yaml

```
@@ -333,3 +333,42 @@ jobs:
       base64-subjects: "${{ needs.combine_hashes.outputs.hashes }}"
       upload-assets: true # Optional: Upload to a new release
       provenance-name: kargo-cli.intoto.jsonl
+  update-homebrew-formula:
+    if: github.event_name == 'release'
+    permissions:
+      contents: write # Needed to push changes
+      pull-requests: write # Needed to create a pull request
+    runs-on: ubuntu-latest
+    needs: [ publish-image, publish-cli ]
+    steps:
+      - name: Checkout bottle Repository
```

Tap and bottle are two different things.

```suggestion
      - name: Checkout tap repository
```